Docker/Dockerfile

```diff
@@ -1,33 +0,0 @@
-ARG registry
```

Comment:
I think we can delete all Docker/ directory because we don't use and support this scripts

Reply:
done